source/NVDAObjects/window/scintilla.py

```diff
@@ -199,7 +199,7 @@ def _getLineOffsets(self,offset):
 			end=self._getOffsetFromPoint(location.right, y)
 			# If this line wraps to the next line,
 			# end is the first offset of the next line.
-			if watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,end)==y:
+			if location.top+watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,end)==y:

```

Comment:
This would be the same as:

```suggestion
			if self._getPointFromOffset(end).y==y:
```

As above, this would mean another extra call (i.e. getting an x coordinate we aren't using. I'd like to have this consistent, so either call _getPointFromOffset twice, or `location.top+watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,offset)`. I have the feeling that using the location to manually calculate screen coordinates from client coordinates might be more error prone, in which the extra window message to get the unused x coordinate is justified.

@Michaeldcurran: It would be good to know what you think here.

Reply:
I do prefer abstraction over manual code, thus using pointFromOffset is my preferred approach, even if it causes one more window message.